sample-apps/rails-sample/config/initializers/judoscale.rb

```diff
@@ -3,5 +3,6 @@
 Judoscale.configure do |config|
   # Open https://judoscale-adapter-mock.requestcatcher.com in a browser to monitor requests
   config.api_base_url = ENV["JUDOSCALE_URL"] || "https://judoscale-adapter-mock.requestcatcher.com"
+  config.rake_task_ignore_regex = /assets:|db:|middleware/
```

Comment:
I don't think we'd run into any by including `middleware` by default, but for now I think I'd rather leave it out. It's not typically run in build processes or prod environments, and it's a quick task as opposed to a task that could take a while.

I don't think our default `rake_task_ignore_regex` needs to be exhaustive. The goal is really to avoid starting the reporter during common build processes.

Reply:
Gotcha, sounds good.